apis/v1alpha1/adminnetworkpolicy_types.go

```diff
@@ -60,7 +60,9 @@ type AdminNetworkPolicySpec struct {
 	// higher precedence, and are checked before rules with higher priority values.
 	// All AdminNetworkPolicy rules have higher precedence than NetworkPolicy or
 	// BaselineAdminNetworkPolicy rules
-	// The behavior is undefined if two ANP objects have same priority.
+	// Every AdminNetworkPolicy should have a unique priority value. If multiple
+	// policies have the same priority, and match the same connection, then it is
+	// implementation-defined which policy's rules get applied to it.
```

Comment:
maybe replace "implementation-defined" with "the policies will be applied in some order"? implementation-defined feels like implementations must tell you an order, but I think the point is to not rely on any specific order here

Reply:
I said "implementation-defined" because I wanted to make it clear that different implementations might do different things. I also didn't want to say anything that could be read as implying that the result will be _predictable_...

Updated... what do you think of the new text?